src/app/components/views/register/register.component.html

```diff
@@ -46,8 +46,22 @@ <h1 class="display-2 fw-bold mb-2">Register</h1>
           />
           <div *ngIf="submitted && password.errors" class="text-danger">
             <div *ngIf="password.errors.required">Password is required</div>
-            <div *ngIf="password.errors.minlength && !password.errors.required">
-              Password should be minimum 6 characters
+            <div *ngIf="!password.errors.required">
+              <div *ngIf="password.value.length < 8">
+                Password should have minimum 8 characters
+              </div>
+              <div *ngIf="password.value.length > 30">
+                Password should have maximum 30 characters
+              </div>
+              <div *ngIf="!passwordStrength.lower.exec(password.value)">
+                Password must have a-z characters
```

Comment:
Change to: Password must have at least one lower case a-z character. Same for upper case and digits